ch04.adoc

```diff
@@ -175,49 +152,38 @@ The `formula_terms` attribute can be used to associate terms in the definitions
 To maintain backwards compatibility with COARDS the use of these attributes is not required, but is strongly recommended.
 Some of the definitions may be supplemented with information stored in the `grid_mapping` variable about the datum used as a vertical reference (e.g. geoid, other geopotential datum or reference ellipsoid; see <<grid-mappings-and-projections>> and <<appendix-grid-mappings>>).
 
-
 [[atm-sigma-coord-ex]]
 [caption="Example 4.3. "]
 .Atmosphere sigma coordinate
 ====
 
 ----
-
 float lev(lev) ;
   lev:long_name = "sigma at layer midpoints" ;
   lev:positive = "down" ;
   lev:standard_name = "atmosphere_sigma_coordinate" ;
   lev:formula_terms = "sigma: lev ps: PS ptop: PTOP" ;
   lev:computed_standard_name = "air_pressure" ;
-
 ----
 
-
 ====
 
 In this example the **`standard_name`** value **`atmosphere_sigma_coordinate`** identifies the following definition from <<parametric-v-coord>> which specifies how to compute pressure at gridpoint **`(n,k,j,i)`** where **`j`** and **`i`** are horizontal indices, **`k`** is a vertical index, and **`n`** is a time index:
-----
 
+----
 p(n,k,j,i) = ptop + sigma(k)*(ps(n,j,i)-ptop)
-
 ----
 
-
-
 The **`formula_terms`** attribute associates the variable **`lev`** with the term **`sigma`**, the variable **`PS`** with the term **`ps`**, and the variable **`PTOP`** with the term **`ptop`**.
-Thus the pressure at gridpoint **`(n,k,j,i)`** would be calculated by
-----
+Thus the pressure at gridpoint **`(n,k,j,i)`** would be calculated by 
```

Comment:
Another bold and monospaced item that probably should only have the latter formatting?

```suggestion
Thus the pressure at gridpoint `(n,k,j,i)` would be calculated by 
```

Reply:
I would agree